kubernetes/crds/machine.sapcloud.io_machinedeployments.yaml

```diff
@@ -60,7 +60,6 @@ spec:
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
           type: string
         metadata:
-          description: Standard object metadata.
```

Comment:
>With k8s version 1.18 metadata.description isn't supported
Can you please give a bit more context on what does it mean? 
Also, do we also need to remove from other crd types, or is it already removed?

Reply:
I wasn't able to check in detail. However, when I tried to deploy this onto k8 1.18 it failed with a similar message - https://github.com/kubernetes/kubernetes/issues/80493. 

However, once I removed it worked well. It probably is an unknown validation field or a deprecated one. Also, i don't think it is meant for meaning anything. 

I checked other CRDs and they don't have this field set so should be fine there.